### ISSUE TITLE
Docker image broken

### ISSUE THREAD

**Issue by Niek** [taskforcesh/taskforce-connector#45]
The docker image is not working, because of PM2 issues:
```bash
PM2 error: TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received type number (23)
```

I worked around by ditching PM2 completely and using this in my docker compose file:
```yaml
command: taskforce --name "${TASKFORCE_CONNECTION}" --passwd "${REDIS_PASSWORD}"
```

**Comment by manast** [comment]
Thanks for reporting, I will take a look at it.

**Comment by marcusglowe** [comment]
This appears to be a pm2 error that was just fixed in 4.4. https://github.com/Unitech/pm2/issues/5189

**Comment by marcusglowe** [comment]
note, i wasn't able to get this to work with an upgraded pm2, probably because of some busted args, so I just removed pm2 and i'm hoping taskforce is stable enough for our local dev needs. Here's the fork: https://github.com/Create-Inc/taskforce-connector
